src/Extensions/TimedScopes/TaskExtensions.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System.Drawing.Drawing2D;
```

Comment:
Drawing2D?

Reply:
Thank you, removed it, must've been added automatically during type guessing :)